### ISSUE TITLE
Snowflake credentials not materializing properly in dbtcloud_environment resource

### ISSUE THREAD

**Issue by boxysean** [dbt-labs/dbtcloud-terraforming#12]
When using library this on a Snowflake project, it spat out:

```
resource "dbtcloud_environment" "terraform_managed_resource_60" {
  credential_id = dbtcloud_credential.terraform_managed_resources_137.id
  dbt_version = "..."
  ...
}
```

However this has two small issues:

1. `dbtcloud_credential` should be `dbtcloud_snowflake_credential` -- because the credential resources are written out as `dbtcloud_snowflake_credential`
2. `dbtcloud_credential.[...].id` should be `dbtcloud_credential.[...].credential_id` -- because the `id` property looks like `24512:2515` rather than a regular ID like `2515`.

**Comment by b-per** [comment]
Thanks! I will fix that shortly.

Not sure why it wasn't raised in the testing I have set up.

**Comment by b-per** [comment]
Fixed in 0.4.1
